server/controllers/events/testfixtures/test-repos/state-rm-multiple-project/exp-output-autoplan.txt

```diff
@@ -0,0 +1,81 @@
+Ran Plan for 2 projects:
+
+1. dir: `dir1` workspace: `default`
+1. dir: `dir2` workspace: `default`
+
+### 1. dir: `dir1` workspace: `default`
+<details><summary>Show Output</summary>
+
+```diff
+
+Terraform used the selected providers to generate the following execution
+plan. Resource actions are indicated with the following symbols:
++ create
+
+Terraform will perform the following actions:
+
+  # random_id.dummy will be created
++ resource "random_id" "dummy" {
+      + b64_std     = (known after apply)
+      + b64_url     = (known after apply)
+      + byte_length = 1
+      + dec         = (known after apply)
+      + hex         = (known after apply)
+      + id          = (known after apply)
+      + keepers     = {}
+    }
+
+Plan: 1 to add, 0 to change, 0 to destroy.
+
+
```

Comment:
Extra line spaces here too

Reply:
This spaces are better to fix here, but it needs fixing many e2e cases. I'll do it later.

https://github.com/runatlantis/atlantis/blob/de953fe25575ac2acd5709fd0dea26f946e3effc/server/events/markdown_renderer.go#L268

Reply:
Ah in another pr you mean? Sounds reasonable to me. It was a nitpick i noticed since you resolved it in another section

Reply:
yes > another pr